pkg/acsengine/engine.go

```diff
@@ -487,12 +487,16 @@ func getParameters(cs *api.ContainerService, isClassicMode bool) (map[string]int
 	return parametersMap, nil
 }
 
+func AddValue(m map[string]interface{}, k string, v interface{}) { addValue(m, k, v) }
```

Comment:
I completely agree with this.. the reason I didn't make that change is because I think there is a lot more work involved once we start looking at the `acsengine` package as a library.. was hoping we could have another PR that cleaned some of that up and offered docs on how `interpolator`'s moving forward could take advantage of the freshly exported `acsengine` functions..

I know it's hacky, and will change if we really want it :) just sharing my thoughts